build-constraints.yaml

```diff
@@ -8255,19 +7556,6 @@ packages:
         - variable-media-field-dhall < 0 # tried variable-media-field-dhall-0.1.0.0, but its *library* requires the disabled package: dhall
         - variable-media-field-dhall < 0 # tried variable-media-field-dhall-0.1.0.0, but its *library* requires the disabled package: variable-media-field
         - variable-media-field-optics < 0 # tried variable-media-field-optics-0.1.0.0, but its *library* requires the disabled package: variable-media-field
-        - vcswrapper < 0 # tried vcswrapper-0.1.6, but its *library* requires base >=4.0.0.0 && < 4.11 and the snapshot contains base-4.19.1.0
-        - vcswrapper < 0 # tried vcswrapper-0.1.6, but its *library* requires containers >=0.5.5.1 && < 0.6 and the snapshot contains containers-0.6.8
-        - vcswrapper < 0 # tried vcswrapper-0.1.6, but its *library* requires mtl >=2.0.1.0 && < 2.3 and the snapshot contains mtl-2.3.1
-        - vcswrapper < 0 # tried vcswrapper-0.1.6, but its *library* requires text >=0.11.1.5 && < 1.3 and the snapshot contains text-2.1.1
-        - vector-circular < 0 # tried vector-circular-0.1.4, but its *library* requires base >=4.11 && < 4.17 and the snapshot contains base-4.19.1.0
```

Comment:
Oops, this one was compatible with GHC 9.2, which is later than what I aimed to exclude. Should I remove it?

Reply:
That might be more consistent then.

Reply:
Removed.